fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -6,27 +6,114 @@ package org.mozilla.fenix.tabstray
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.google.accompanist.pager.rememberPagerState
+import kotlinx.coroutines.launch
+import org.mozilla.fenix.compose.Divider
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.PrimaryButton
 import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
  * Top-level UI for displaying the Tabs Tray feature.
  */
+@OptIn(ExperimentalPagerApi::class)
```

Comment:
nit: I think adding a TODO here to switch from this library to the official compose version once we are on 1.4 could be useful if you think there is a chance we will continue to use it.

Reply:
Is this component in 1.4? I may have missed that

Reply:
According to the [accompanist README](https://github.com/google/accompanist#-pager-soon-to-be-deprecated---upstreamed-in-compose-14)

Reply:
I opened a follow-up ticket and added it to the mega meta